seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/JettyService.java

```diff
@@ -88,14 +94,63 @@ public class JettyService {
     public JettyService(NodeEngineImpl nodeEngine, SeaTunnelConfig seaTunnelConfig) {
         this.nodeEngine = nodeEngine;
         this.seaTunnelConfig = seaTunnelConfig;
+        this.server = new Server();
+        configureServer();
+    }
+
+    private void configureServer() {
```

Comment:
So we don't support enable https and http at same time. Please add verify to make sure user not open https/http both in config.